jake/pip/pip.py

```diff
@@ -24,19 +24,25 @@
 
 class Pip():
   """pip.py gets installed pip dependencies"""
-  def __init__(self, targets=None):
+  def __init__(self, targets=None, requirements_file_path=None):
     self._log = logging.getLogger('jake')
     self._format = "pypi"
     self._working_set = pkg_resources.working_set
+    self.requirements_file_path = requirements_file_path
     if targets:
       self._working_set = pkg_resources.WorkingSet(ast.literal_eval(targets))
-    self._coords = self.generate_dependencies()
+    self.__coords = self.generate_dependencies()
```

Comment:
Is this change just to make it clear the `__coords` dunder is different? 

I also found it a little confusing that the original code sets the `self._coords`.

Reply:
I made the PR at least a week ago, but I don't remember any intention for this.
This might be just a mistake. I'll revert this line